addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -114,16 +114,27 @@ publicWidget.registry.popup = PopupWidget;
 // Prevent bootstrap to prevent scrolling and to add the strange body
 // padding-right they add if the popup does not use a backdrop (especially
 // important for default cookie bar).
+
+const _baseShowElement = $.fn.modal.Constructor.prototype._showElement;
+$.fn.modal.Constructor.prototype._showElement = function (relatedTarget){
+    _baseShowElement.apply(this, relatedTarget);
```

Comment:
`apply` expects a list of arguments to use, you give a simple value. Not that the code which already exists is currently wrong too :fearful: 

`_baseSetScrollbar.apply(this, ...arguments);` -> makes no sense, it should be:
`_baseSetScrollbar.apply(this, arguments);`
or
`_baseSetScrollbar.call(this, ...arguments);`

Can you fix those at the same time? Use `arguments` for your new method override too.